samples/streaming/consumer/src/main.rs

```diff
@@ -46,11 +48,20 @@ async fn stream_images(
         }
         let media_content = opt_media.unwrap().media_content;
```

Comment:
Is it possible for this unwrap() to panic?

Reply:
It's okay if it panics.